src/Model/BaseManager.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\Doctrine\Model;
+
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Doctrine\Common\Persistence\ObjectManager;
+use Doctrine\Common\Persistence\ObjectRepository;
+
+/**
+ * @author Hugo Briand <briand@ekino.com>
+ */
+abstract class BaseManager implements ManagerInterface
+{
+    /**
+     * @var ManagerRegistry
+     */
+    protected $registry;
+
+    /**
+     * @var string
+     */
+    protected $class;
+
+    /**
+     * @param string          $class
+     * @param ManagerRegistry $registry
+     */
+    public function __construct($class, ManagerRegistry $registry)
+    {
+        $this->registry = $registry;
+        $this->class = $class;
+    }
+
+    /**
+     * @return ObjectManager
+     */
+    public function getObjectManager()
+    {
+        $manager = $this->registry->getManagerForClass($this->class);
+
+        if (!$manager) {
+            throw new \RuntimeException(sprintf(
+                'Unable to find the mapping information for the class %s.'
+                .' Please check the `auto_mapping` option'
+                .' (http://symfony.com/doc/current/reference/configuration/doctrine.html#configuration-overview)'
+                .' or add the bundle to the `mappings` section in the doctrine configuration.',
+                $this->class
+            ));
+        }
+
+        return $manager;
+    }
+
+    public function getClass()
```

Comment:
No phpdoc comments at all on functions under, shouldn't comments be added?

Reply:
Comment is here:

https://github.com/sonata-project/sonata-doctrine-extensions/pull/82/files#diff-be130aff7fc129752dd05b9b8b99a7eaR26

Reply:
Saw it after the comment, what about `inheritdoc` I think I saw somewhere we do not use that?

Reply:
We don't use this anymore :(

https://github.com/sonata-project/dev-kit/issues/345

Reply:
that is it, I knew I saw it somewhere, ty for the link :smile: